src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -108,7 +125,7 @@ export default class ScopeModel {
      * @param radarTargetModel {RadarTargetModel}
      * @return result {array} [success of operation, system's response]
      */
-    acceptHandoff(radarTargetModel) {
+    acceptHandoff(/* radarTargetModel */) {
```

Comment:
could you add a comment (maybe even in the `@class` docblock) about why so many params are commented out?

Reply:
Generally they're for lint since they are dummies that aren't actually used yet. I'll add a note for it!

Reply:
done